packages/cache/src/CacheStore.spec.tsx

```diff
@@ -81,6 +87,19 @@ describe('CacheStore', () => {
     expect(cacheStore.getError(errorCache(2))).toBeUndefined()
   })
 
+  it('should take a key and remove the cached data for that key', async () => {
+    try {
+      cacheStore.suspend(successCache(1))
+    } catch (promiseToSuspense) {
+      await promiseToSuspense
+    }
+    expect(cacheStore.getData(successCache(1))).toBeTruthy()
```

Comment:
In my opinion, other method could be clear than expect(x).toBeTruthy for this case

```suggestion
    expect(cacheStore.getData(successCache(1))).toBeTruthy()
```

Reply:
I agree your opinion! fixed it https://github.com/toss/suspensive/pull/1140/commits/53e03048d19033fe2f6b6063ee0dcf8e3e7beed7